pyaedt/modeler/cad/Primitives3D.py

```diff
@@ -57,9 +57,9 @@ def create_box(self, position, dimensions_list, name=None, matname=None):
         Parameters
         ----------
         position : list
-            Anchor point for the box in Cartesian ``[x, y, z]`` coordinates.
+            Anchor point for the box in Cartesian``[x, y, z]`` coordinates.
```

Comment:
Why were spaces removed from this description and the next? They should be there for readability of the sentence.